charts/consul/values.yaml

```diff
@@ -3248,7 +3248,7 @@ apiGateway:
   # The name (and tag) of the Envoy Docker image used for the
   # apiGateway. For other Consul compoenents, imageEnvoy has been replaced with Consul Dataplane.
   # @default: envoyproxy/envoy:<latest supported version>
-  imageEnvoy: "envoyproxy/envoy:v1.25.1"
+  imageEnvoy: "envoyproxy/envoy:v1.25.11"
```

Comment:
v1.25.x is the latest version that I saw for the legacy API Gateway. I'm not sure how/why the value on this branch is so out-of-date, but I've bumped it to the latest for consistency with the other branches.